streamwebs_frontend/streamwebs/templates/streamwebs/resources/resources_tutorial_videos.html

```diff
@@ -0,0 +1,42 @@
+{% extends "streamwebs/base.html" %}
+{% load staticfiles %}
+{% load i18n %}
+
+{% block content %}
+  <div class="container">
+    {% if messages %}
+        {% for message in messages %}
+        <strong>{{ message }}</strong>
+        {% endfor %}
+    {% endif %}
+
+    <h1 align="center">{% trans "Tutorial Videos" %}</h1>
+
+    <table class="bordered centered responsive-table">
+      <thead>
+        <tr>
+            <th class="tutorial-th" data-field="name">Name</th>
+            <th class="tutorial-th" data-field="Thumbnail">Video</th>
+        </tr>
+      </thead>
+    <tbody>
+      {% for video in data %}
+        <tr>
+          <td>
+            <h4>
+              {{ video.name }}
+            </h4>
+          </td>
+          <td>
+            <video width="320" height="240" controls>
```

Comment:
Not sure what the size of the video player shouid be. I have it set to 320x240, but can be fullscreened